file-formats/sush/type/zif_aff_sush_v1.intf.abap

```diff
@@ -0,0 +1,186 @@
+      "! Maintenance Mode.
+      "! $values {@link zif_aff_sush_v1.data:co_usob_status_flag}
+      "! $required
+      okflag type ty_usob_status_flag,
+    end of ty_general .
+
+  types:
+    "! <p class="shorttext">Authorization Value</p>
+    "! Authorization Value
+    ty_val type c length 40.
+
+  types:
+    "! <p class="shorttext">Default Authorization Values</p>
+    "! Default Authorization Values
+    begin of ty_value,
+      "! <p class="shorttext">From</p>
+      "! From Value
+      "! $showAlways
+      low  type ty_val,
+      "! <p class="shorttext">To</p>
+      "! To Values
+      "! $showAlways
+      high type ty_val,
+    end of ty_value .
+
+  types:
+    "! <p class="shorttext">Authorization Fields</p>
+    "! Authorization Fields
+    ty_values  type standard table of ty_value with default key.
+
+  types:
+    "! <p class="shorttext">Authorization Field</p>
+    "! Authorization Field
+    ty_field type c length 10.
+
+  types:
+    "! <p class="shorttext">Authorization Field Details</p>
+    "! Authorization Field Details
+    begin of ty_fldtablestruc,
+      "! <p class="shorttext">Field</p>
+      "! Name of Authorization Field
+      "! $required
+      field  type ty_field,
+      "! <p class="shorttext">Values</p>
+      "! Name of Authorization Field Value
+      values type ty_values,
+    end of ty_fldtablestruc .
+
+  types:
+    "! <p class="shorttext">Authorization Fields</p>
+    "! Authorization Fields
+    ty_authorization_fields  type standard table of ty_fldtablestruc with default key.
+
+  types:
+    "! <p class="shorttext">Authorization Objects Details</p>
+    "! Authorization Objects Details
+    begin of ty_objtablestruc,
+      "! <p class="shorttext">Object</p>
+      "! Name of Authorization Object
+      "! $required
+      object type c length 10,
+      "! <p class="shorttext">Object Description</p>
+      "! Authorization Object Description
+      "! $showAlways
+      ttext  type zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Maintenance Status</p>
+      "! Maintenance Status
+            "! $values {@link zif_aff_sush_v1.data:co_okflag}
+      "! $default {@link zif_aff_sush_v1.data:co_okflag.y_flag}
+      "! $showAlways
+      okflag type ty_okflag,
+      "! <p class="shorttext">Fields</p>
+      "! Authorization Fields.
+      "! $showAlways
+      fields type ty_authorization_fields,
+
+    end of ty_objtablestruc .
+
+  types:
+  "! <p class="shorttext">Authorization Objects</p>
+  "! Authorization Objects
+    ty_authorization_objects type standard table of ty_objtablestruc with default key.
+
+  types:
+    "! <p class="shorttext">SUSH Object Type</p>
+    "! Object type SUSH
+    begin of  ty_main,
+      "! $required
+      format_version        type zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                type zif_aff_types_v1=>ty_header_60_src,
```

Comment:
`zif_aff_types_v1=>ty_header_60_src` is wrong here, because it shall be used for source code objects, only (for technical reasons). I know this is confusing :(

As far as I know you support ABAP language versions, `default` and `cloudDevelopment`. If this is the case, please use `zif_aff_types_v1=>ty_header_60_cloud`

```suggestion
      header                type zif_aff_types_v1=>ty_header_60_cloud,
```